firmware/ventilator-controller-stm32/Core/Test/Pufferfish/Util/Containers/EnumMap.txt

```diff
@@ -0,0 +1,357 @@
+SCENARIO('The EnumMap is constructed from initializer list works corretly') 
+  GIVEN('An EnumMap constructed with capacity 11, has {1, 1}, {2,2}, {3,3}, {4,4} items') 
+    WHEN('The size method is called')
+      THEN('The size method returns size as 4')
+
+    WHEN('The avaliable method is called ')
+      THEN('The avaliable method reports that 7 items are avlaible ')
```

Comment:
It would be less ambiguous to say that there are 7 spaces available (since the map has 4 items and 7 spaces in it)